docker-compose/docker-compose.yaml

```diff
@@ -499,6 +500,8 @@ services:
       - '/dev/disk/:/dev/disk:ro'
       # Uncomment to enable container monitoring on MacOS
       # - '/var/run/docker.sock:/var/run/docker.sock:ro'
+    devices:
+      - '/dev/kmsg'
```

Comment:
all flags are documented here https://github.com/google/cadvisor/blob/master/README.md#quick-start-running-cadvisor-in-a-docker-container

these two flags were not in the readme when MI was invented in the first place (I think) and that's how we missed them

https://github.com/google/cadvisor/issues/2150
https://github.com/google/cadvisor/pull/2545